lab_1_classify_by_unigrams/main.py

```diff
@@ -3,6 +3,8 @@
 
 Language detection
 """
+
```

Comment:
revert this lines

Reply:
done

Reply:
No, or am I the only one who sees this change in your PR?

Reply:
i reverted the line